experimental/dds/tree2/src/feature-libraries/editable-tree-2/editableTreeTypes.ts

```diff
@@ -150,6 +167,23 @@ export interface TreeNode extends TreeEntity<TreeNodeSchema> {
 	readonly type: TreeNodeSchemaIdentifier;
 
 	[boxedIterator](): IterableIterator<TreeField>;
+
+	/**
+	 * Subscribe to the next change that affects this node's children.
```

Comment:
Nit: rather than duplicating the docs here and on the symbol, might be worth having one `@inheritDoc` from the other. Makes the intellisense slightly worse due to lack of `@inheritDoc` support, but in this case I'm not sure it matters too much. Especially if you have the symbol inherit from this.

Reply:
I previously tried to inherit the method doc from the symbol. This does not work, because the symbol is not publicly exposed (but the method "is"). Do you know how to inherit the symbol doc from the method? I don't know how to reference the method in TS doc, since the property is a symbol key, not a string.

Reply:
Oh, urg. I'm not sure if it's possible.

This page on TSDoc's site gives an example that it claims works, though I don't know how confident I am in it. Still, might be worth a try. I guess it would be something like:
```
{@inheritDoc TreeNode.[onNextChange]}
```

Or maybe it even needs the paren?
```
{@inheritDoc TreeNode.([onNextChange])}
```

If not, I'm sad.

Reply:
I did your first suggestion. I'm not sure how well it works in mouse-hover, but it's enough for internal dev to find the information they need.